tests/ui/octal_escapes.stderr

```diff
@@ -0,0 +1,67 @@
+error: octal-looking escape in string literal
+  --> $DIR/octal_escapes.rs:4:17
+   |
+LL |     let _bad1 = "/033[0m";
```

Comment:
This looks wrong?

Reply:
From what I understand, the ui-test machinery "normalizes" its output, and one of the steps is to convert backslashes to slashes.

There's an even more confusing normalization on line 37/43/47, where in the real output there would be *three* backslashes instead of two slashes.

Reply:
Interesting :thinking: @Manishearth Do you know if this is caused by the normalizing process of completest? :upside_down_face:

Reply:
No idea! I don't really understand the internals of compiletest

Reply:
https://github.com/Manishearth/compiletest-rs/blob/master/src/runtest.rs#L2636 looks like the culprit.

Reply:
Alright, that is something that should probably be fixed upstream(rust-lang/rust repo) and then updated in compiletest-rs. That shouldn't block this PR, though :upside_down_face:. Thanks for pointing out the specific lines @birkenfeld.

Reply:
I'm curios. Does the test pass if you change it from `/` to `\`? Maybe you can try it locally.

Reply:
It doesn't; it seems that the blessed output is not normalized again before comparison.